src/Plugins/DBFTPlugin/Consensus/ConsensusService.OnMessage.cs

```diff
@@ -98,6 +98,9 @@ private void OnPrepareRequestReceived(ExtensiblePayload payload, PrepareRequest
             // around 2*15/M=30.0/5 ~ 40% block time (for M=5)
             ExtendTimerByFactor(2);
 
+            onPrepareReceivedTime = TimeProvider.Current.UtcNow;
+            onPrepareBlockIndex = message.BlockIndex;
+
```

Comment:
Revert these changes, they are not a part of this PR. This PR must only contain the logic that moves dBFT from ProtocolConfig-based MSPerBlock to Policy-based value.

Reply:
It was originally a part of https://github.com/neo-project/neo/pull/3637, so let's keep this logic there.